src/applications/edu-benefits/utils/labels.jsx

```diff
@@ -179,6 +179,18 @@ export const benefitsLabelsUpdate = {
       </a>
     </p>
   ),
+  chapter35: (
+    <p>
+      Survivors' and Dependents’ Education Assistance (DEA, Chapter 35 )<br />
+      <a
```

Comment:
This could likely be a `va-link` component - https://design.va.gov/storybook/?path=/docs/components-va-link--default

Reply:
We'll be solving that issue in another ticket.